test/e2e/baseline_test.go

```diff
@@ -909,3 +910,37 @@ func PromDiff(t test.Failer, prom prometheus.Instance, cluster cluster.Cluster,
 
 	}
 }
+
+func TestServiceRestart(t *testing.T) {
```

Comment:
This case fails and needs to be fixed.

Reply:
donot we already have a restart case?

Reply:
> donot we already have a restart case?

We have `Kmesh daemon restart` case and this is the restart of applications.